### PR TITLE
feat(demos): add todomvc demo

### PR DESCRIPTION
**Pull request by mrpelz**
Hey @clebert, how about placing the todo-MVC here instead of a separate repo?

### CODE REVIEW

File: packages/demos/src/todomvc/footer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import {FeatureAppDefinition} from '@feature-hub/core';
+import {ReactFeatureApp} from '@feature-hub/react';
+import * as React from 'react';
+import {TodoManagerV1} from '../todo-manager';
+import {TodoMvcFooter} from './todomvc-footer';
+
+interface Dependencies {
```
Comment (clebert) [inline packages/demos/src/todomvc/footer/index.tsx]: export is missing?

File: packages/demos/src/todomvc/footer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  'test:todomvc-todo-manager': TodoManagerV1;
```
Comment (clebert) [inline packages/demos/src/todomvc/footer/index.tsx]: readonly

File: packages/demos/src/todomvc/header/index.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import {FeatureAppDefinition} from '@feature-hub/core';
+import {DomFeatureApp} from '@feature-hub/react';
+import {TodoManagerV1} from '../todo-manager';
+import {TodoMvcHeader} from './todomvc-header';
+
+interface Dependencies {
```
Comment (clebert) [inline packages/demos/src/todomvc/header/index.ts]: export & readonly

File: packages/demos/src/todomvc/main/index.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import {FeatureAppDefinition} from '@feature-hub/core';
+import {ReactFeatureApp} from '@feature-hub/react';
+import * as React from 'react';
+import {TodoManagerV1} from '../todo-manager';
+import {TodoMvcMain} from './todomvc-main';
+
+interface Dependencies {
```
Comment (clebert) [inline packages/demos/src/todomvc/main/index.tsx]: export & readonly

File: packages/demos/src/todomvc/todo-manager/index.ts [hunk 5]
```diff
@@ -88,14 +78,16 @@ class TodoManagerV1Impl implements TodoManagerV1 {
...
-  private set todos(todos: Todo[]) {
-    this._todos = todos;
-
+  private notifyListeners(): void {
     this.listeners.forEach(listener => listener());
   }
 
-  private get todos(): Todo[] {
-    return this._todos;
+  private getNextId(): string {
```
Comment (clebert) [inline packages/demos/src/todomvc/todo-manager/index.ts]: `nextId`

File: packages/demos/src/todomvc/main/index.tsx [hunk 6]
```diff
@@ -4,15 +4,15 @@ import * as React from 'react';
 import {TodoManagerV1} from '../todo-manager';
 import {TodoMvcMain} from './todomvc-main';
 
-interface Dependencies {
+export interface MainFeatureServices {
   'test:todomvc-todo-manager': TodoManagerV1;
```
Comment (clebert) [inline packages/demos/src/todomvc/main/index.tsx]: readonly

Comment (KingHenne) [reply packages/demos/src/todomvc/main/index.tsx]: done

**Pull request merged**